code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_5.swift

```diff
@@ -3,25 +3,12 @@ import SwiftUI
 import RevenueCat
 import RevenueCatUI
 
-struct App: View {
+struct YourPaywall: View {
 
     var body: some View {
-        VStack {
-            ScrollView {
-                VStack {
-                    // Your custom paywall design content
-                }.frame(maxWidth: .infinity)
-            }
-
-            Spacer()
-
-            PaywallView(mode: .condensedCard) // or .card
-        }
-        .frame(maxWidth: .infinity, maxHeight: .infinity)
-        .background(
-            Color(white: 0.8)
-                .edgesIgnoringSafeArea(.all)
-        )
+        ScrollView {
+            // Your custom paywall design content
+        }.paywallFooter()
```

Comment:
Nit: move to its own line?
```suggestion
        }
        .paywallFooter()
```